package.json

```diff
@@ -41,8 +41,8 @@
     "mocha": "^5.0.0",
     "nyc": "^13.2.0",
     "pre-commit": "^1.2.2",
-    "react": "^15.4.2",
-    "react-dom": "^15.4.2",
+    "react": ">=15",
+    "react-dom": ">=15",
```

Comment:
In the future it's fine and ideal for devDeps to point to a more specific version (while peerDeps uses >=)